### ISSUE TITLE
Playerbase Development Details of Activity Level Change

### ISSUE THREAD

**Issue by Exophobias** [plan-player-analytics/Plan#2859]
### I would like to be able to..

On the Playerbase > Overview page, I would like to be able to select a date-range and see what players had their activity level changed during X-Y period. 

### Is your feature request related to a problem? Please describe.

N/A

**Comment by AuroraLS3** [comment]
This is currently being planned after the frontend rewrite is complete :)

**Comment by Exophobias** [comment]
Sweet, is there a beta for the new front-end? I see the beta-tag on some issues

**Comment by AuroraLS3** [comment]
Yes set `Plugin.Frontend_BETA` to true in the config and it'll show up

**Comment by AuroraLS3** [comment]
The Frontend BETA tagged issues are tickets that need https://github.com/plan-player-analytics/Plan/issues/2260 to be completed first or are related to it
